.github/workflows/Packages - EdFi.Ods.Extensions.SK.yml

```diff
@@ -0,0 +1,129 @@
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+env:
+  INFORMATIONAL_VERSION: "6.1"
+  BUILD_INCREMENTER: "4"
+  CONFIGURATION: "Release"
+  CURRENT_BRANCH:  ${{ GITHUB.REF_NAME }}
+  AZURE_ARTIFACT_URL: "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
+  AZURE_ARTIFACT_NUGET_KEY: ${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }}
+  VSS_NUGET_EXTERNAL_FEED_ENDPOINTS : '{"endpointCredentials": [{"endpoint": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json","password": "${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }}"}]}'
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - name: Setup .NET
+      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # v2.1.0
+      with:
+        dotnet-version: 6.0.x
+    - name: Support longpaths
+      run: git config --system core.longpaths true
+    - name: Checkout Starter-Kit-SEA-Modernization
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Starter-Kit-SEA-Modernization
+          path: Starter-Kit-SEA-Modernization/
+    - name: Checkout Ed-Fi-ODS-Implementation
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
+          path: Ed-Fi-ODS-Implementation/
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+          $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+          $is_pull_request_branch = 'False'
+          $is_pull_request_branch = git ls-remote --heads origin ${{ env.CURRENT_BRANCH }} | Select-String -Pattern $patternName -SimpleMatch -Quiet
+          if ($is_pull_request_branch -eq $true) {
+            git fetch origin ${{ env.CURRENT_BRANCH }}
+            git checkout ${{ env.CURRENT_BRANCH }}
+          }
+    - name: Checkout Ed-Fi-ODS
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
+          path: Ed-Fi-ODS/
+    - name: Is pull request branch exists in Ed-Fi-ODS
+      working-directory: ./Ed-Fi-ODS/
+      shell: pwsh
+      run: |
+          $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+          $is_pull_request_branch = 'False'
+          $is_pull_request_branch = git ls-remote --heads origin ${{ env.CURRENT_BRANCH }} | Select-String -Pattern $patternName -SimpleMatch -Quiet
+          if ($is_pull_request_branch -eq $true) {
+            git fetch origin ${{ env.CURRENT_BRANCH }}
+            git checkout ${{ env.CURRENT_BRANCH }}
+          }
+    - name: Register Ed-Fi Azure Artifacts Package Source
```

Comment:
Good catch on finding how we needed to add this so that it could restore packages from the Azure Artifacts feed since it is not specified in the nuget config by default for this build

Reply:
Thanks